docs/calendar-draft.md

```diff
@@ -334,6 +334,7 @@ The following table describes these semantics.  Option 5 is not shown because th
 | T.now.isoDate() | N/A | N/A | N/A | N/A | Full ISO |
 | absolute.inTimeZone() | Full ISO | Explicit | Partial ISO | Environ. | Explicit |
 | absolute.inZoneISO() | N/A | N/A | N/A | N/A | Full ISO |
+| date.getMonthDay() | Full ISO | Inherit | Explicit | Environ. | Inherit |
```

Comment:
I think in options 1 and 4 this is "Inherit" as well, no? I can only imagine that we'd want to do something different than "Inherit" in options 3 and 5.

Reply:
Clarified (See below)